<define-tag description>denial of service</define-tag>
<define-tag moreinfo>
<p>It was discovered that python-django, a high-level Python web
develompent framework, is prone to a denial of service vulnerability
via large passwords.</p>

<p>A non-authenticated remote attacker could mount a denial of service by
submitting arbitrarily large passwords, tying up server resources in
the expensive computation of the corresponding hashes to verify the
password.</p>

<p>For the oldstable distribution (squeeze), this problem has been fixed in
version 1.2.3-3+squeeze8.</p>

<p>For the stable distribution (wheezy), this problem has been fixed in
version 1.4.5-1+deb7u4.</p>

<p>For the unstable distribution (sid), this problem has been fixed in
version 1.5.4-1.</p>

<p>We recommend that you upgrade your python-django packages.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2013/dsa-2758.data"
# $Id$
